parm/atm/obs/config/lgetkf_amsua_n19.yaml

```diff
@@ -0,0 +1,73 @@
+obs space:
+  name: amsua_n19
+  distribution:
+    name: Halo
+    halo size: 1250e3
+  obsdatain:
+    obsfile: $(OBS_DIR)/$(OBS_PREFIX)amsua_n19.$(OBS_DATE).nc4
+  obsdataout:
+    obsfile: $(DIAG_DIR)/diag_amsua_n19_lgetkf.$(OBS_DATE).nc4
+  io pool:
+    max pool size: 1
+  simulated variables: [brightness_temperature]
+  channels: 4-6,9-14
+obs operator:
+  name: CRTM
+  Absorbers: [H2O,O3]
+#   Clouds: [Water, Ice]
+#   Cloud_Fraction: 1.0
+  obs options:
+    Sensor_ID: amsua_n19
+    EndianType: little_endian
+    CoefficientPath: crtm/
+obs error:
+  covariance model: diagonal
+obs filters:
+- filter: Bounds Check
+  filter variables:
+  - name: brightness_temperature
+    channels: 4-6,9-14
+  minvalue: 100.0
+  maxvalue: 500.0
+- filter: Background Check
+  filter variables:
+  - name: brightness_temperature
+    channels: 4-6,9-14
+  threshold: 3.0
+- filter: Domain Check
+  filter variables:
+  - name: brightness_temperature
+    channels: 4-6,9-14
+  where:
+  - variable:
+      name: scan_position@MetaData
+    minvalue: 4
+    maxvalue: 27
+  - variable:
+      name: brightness_temperature_1@ObsValue
+    minvalue: 50.0
+    maxvalue: 550.0
+  - variable:
+      name: brightness_temperature_2@ObsValue
+    minvalue: 50.0
+    maxvalue: 550.0
+  - variable:
+      name: brightness_temperature_3@ObsValue
+    minvalue: 50.0
+    maxvalue: 550.0
+  - variable:
+      name: brightness_temperature_4@ObsValue
+    minvalue: 50.0
+    maxvalue: 550.0
+  - variable:
+      name: brightness_temperature_6@ObsValue
+    minvalue: 50.0
+    maxvalue: 550.0
+  - variable:
+      name: brightness_temperature_15@ObsValue
+    minvalue: 50.0
+    maxvalue: 550.0
+obs localizations:
```

Comment:
are the only differences here in these YAMLs the localization? We should figure out how to have this come from another YAML and include it in the YAML generation step.

Reply:
Here is how `lgetkf_amsua_n19.yaml` differs, at present, from `amsua_n19.yaml`
- add `obs localization` section
- add `distribution halo` section
- remove `obs bias` section
- greatly simplify `obs filter` section

At this point I am only running plumbing tests.   What you suggest makes sense as we move beyond the basic plumbing tests.   I would expect the observation yamls to be similar between JEDI-VAR and JEDI-ENS.   This is true in GSI-based DA apart from a few namelist changes we make in eobs.

Reply:
@CoryMartin-NOAA , I thought a bit more about this.  I agree with you.  As we add more obs yamls to GDASApp, we don't want to maintain VAR and ENS obs yamls which are basically the same.   Some options.  (1) As you say, extend the yaml generation script to combine the JEDI-VAR obs yamls with JEDI-ENS specific information (localization and distribution).   (2) Run hofx prior to JEDI-ENS.   This is akin to operations with eobs running before eupd.   With (2) I'm not sure if we need localization and distribution in hofx.  So it's possible (2) still needs (1).   I have not tried running JEDI-ENS with precomputed hofx input.  

However we decide to handle this, I think it would be good to get this PR into develop as a zeroth-order cycling capability.  Subsequent PRs can improve upon this baseline.